projects/js-packages/connection/components/connection-error-notice/styles.module.scss

```diff
@@ -47,8 +47,6 @@
 
 .link {
 	margin-left: auto;
-	padding-left: 8px;
+	padding-left: var( --spacing-base );
 	text-decoration: underline;
```

Comment:
You can remove this one. It's not needed since `Button` applies it

Reply:
good catch! confirmed. 

Correcting it now